### PR TITLE
Add Release Schedule

### PR DESCRIPTION
**Pull request by fhemberger**
Pulls data from https://github.com/nodejs/Release

Fixes: #1889

### CODE REVIEW

**Review by ZYSzys — APPROVED** [review]
LGTM.
But IMHO, I wonder if it's better to put this section into `about/releases.md` since this is release related ?

File: layouts/about-release-schedule.hbs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <div id="main">
+    <div class="container has-side-nav">
+
+      {{> navigation key='about'}}
+
+      <article>
+        <h1>{{ title }}</h1>
+
+        <p>
+          <img alt="{{ title }}" src="https://raw.githubusercontent.com/nodejs/Release/master/schedule.png">
```
Comment (richardlau) [inline layouts/about-release-schedule.hbs:17]: FYI the PNG might get replaced by SVG (https://github.com/nodejs/Release/pull/380). 

The image is generated from `schedule.json` (see commit message in https://github.com/nodejs/Release/commit/e82fc16b222555415948abdf728a8ee8f304dced). Would it make sense to generate it in the deploy step?

**Comment by richardlau** [comment]
cc @nodejs/release

File: layouts/about-release-schedule.hbs (hunk 1 above)
Comment (fhemberger) [reply layouts/about-release-schedule.hbs:17]: As soon as the PR is merged, I'll switch to the SVG instead.

IMHO it doesn't make sense to be generated on each website update. The release schedule only changes only every couple of months and is updated first in https://github.com/nodejs/Release/ anyway.

**Comment by fhemberger** [comment]
@ZYSzys The "Release" page explains more what kind of changes land in version numbers. I'm currently not sure if we need this explanation anyway or if it can be replaced entirely by the release schedule.

Feedback is appreciated.

**Comment by ZYSzys** [comment]
I didn't mean replacing the "Release" page with the release schedule, I'm saying can we just add a section called `schedule` into the "Release" page simply ?

**Comment by Trott** [comment]
I agree with @fhemberger's suspicion that https://nodejs.org/en/about/releases/ can be deleted entirely or replaced with the schedule here. Most of it is not very informative.

**Comment by fhemberger** [comment]
Ok, replaced the old "Releases" page entirely with the release schedule. The information wasn't really helpful anyway. PTAL.

**Review by Trott — APPROVED** [review]

**Review by marswong — APPROVED** [review]

File: locale/uk/about/releases.md [hunk 2]
```diff
@@ -1,81 +1,18 @@
...
+  pending: 'Pending'
+columns:
+  - 'Release'
+  - 'Status'
+  - 'Codename'
+  - 'Initial Release'
+  - 'Active LTS Start'
+  - 'Maintenance LTS Start'
+  - 'End-of-life'
+schedule-footer: Dates are subject to change.
```
Comment (vsemozhetbyt) [inline locale/uk/about/releases.md]:
```yaml
---
layout: about-release-schedule.hbs
title: Релізи
statuses:
  maintenance: 'Підтримуваний LTS'
  active: 'Активний LTS'
  current: 'Поточний'
  pending: 'Майбутній'
columns:
  - 'Резіз'
  - 'Статус'
  - 'Кодова назва'
  - 'Початковий реліз'
  - 'Початок активного LTS'
  - 'Початок підтримуваного LTS'
  - 'Кінець підтримки'
schedule-footer: Дати можуть змінитись.
---
```

**Comment by fhemberger** [comment]
Thanks everybody!

**Pull request merged**